.github/workflows/build-test-deploy.yml

```diff
@@ -1100,7 +1100,7 @@ jobs:
           installCommand: |
             nix-channel --add ${{ env.NIX_NIXPKGS_CHANNEL }} nixpkgs
             nix-channel --update
-            nix-env -iA nixpkgs.cachix nixpkgs.nodejs
+            nix-env -iA nixpkgs.cachix nixpkgs.nodejs nixpkgs.glib
```

Comment:
should this be reverted?

Reply:
@KaiserKarel I recall you asking about appending this change but can't seem to find the message

Reply:
Yeah that should be reverted now